server/container/guice/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/JMAPCommonModule.java

```diff
@@ -96,4 +99,11 @@ protected void configure() {
                 accessTokenAuthenticationStrategy,
                 queryParameterAuthenticationStrategy);
     }
+
+    @ProvidesIntoSet
+    InitializationOperation workQueue(MailSpool instance) {
```

Comment:
why is this in jmap? It does not seem related...

Reply:
The sin of this class (MailSpool) is generating the mailQueue within the constructor.

The asumption of some code is that I can call the MailQueueFactory within it, which means I don't need to init the MailQueue.

Brittle assumption that I break (as MQ depends on the rabbitChannel pool that should be initialized).

To be sum up, I find it pretty nice that we use the init stage of our app to create connections...